### PR TITLE
Update CLI Quickstart to Use Dependency Manager

### PR DESCRIPTION
**Pull request by chasefleming**
#731

### CODE REVIEW

**Review by ianthpun — APPROVED** [review]

File: docs/build/getting-started/quickstarts/flow-cli.md [hunk 1]
```diff
@@ -19,144 +19,40 @@ For other ways of installing, please [refer to the installation guide](../../../
...
-
-The Flow Emulator is a local, lightweight, and standalone version of the Flow blockchain. It's designed to provide developers with a local environment for testing and development purposes without having to interact with the `mainnet` or `testnet`. This makes it easier and faster to develop, test, and debug smart contracts and apps.
-
-In order to use it, let's update our project configuration. 
+This will add the `HelloWorld` contract and any of its dependencies to an `imports` directory in your project (you can gitignore this directory). It will also add any dependencies to your `flow.json` file.
 
-Let's create a local version of the HelloWorld contract. To do that, we'll again use the `generate` subcommand of the CLI to create a boilerplate contract. Except this time, we'll generate a contract file at `cadence/contracts/HelloWorld.cdc`.
+During the install, it will also prompt you as to which account you'd like to have the contract deployed to. For this tutorial, you can select the default `emulator-account`. You can then interact with them in the local emulator or on any other network just as you would any contract you've written locally and deployed.
```
Comment (gregsantos) [inline docs/build/getting-started/quickstarts/flow-cli.md:53]:
```suggestion
During the install you'll be prompted to specify which account to deploy the contracts to. For this tutorial, you can select the default `emulator-account`.
```

File: docs/build/getting-started/quickstarts/flow-cli.md [hunk 2]
```diff
@@ -19,144 +19,40 @@ For other ways of installing, please [refer to the installation guide](../../../
...
 
-```
-flow generate contract HelloWorld
-```
-
-Now let's update the contract to have a `greeting` variable and a `changeGreeting` function that will allow us to change the greeting.
+If you want to see what the contract looks like, you can open it up and you will see this:
```
Comment (gregsantos) [inline docs/build/getting-started/quickstarts/flow-cli.md:55]:
```suggestion
Opening the contract file in your editor you should see the following:
```

File: docs/build/getting-started/quickstarts/flow-cli.md [hunk 3]
```diff
@@ -175,87 +71,58 @@ access(all) contract HelloWorld {
...
-We're also going to change the imports of our script so that there are no hardcoded network specific addresses. The CLI can figure out how to interact with the network (i.e. emulator, testnet or mainnet) based on our configuration.
+To deploy the `HelloWorld` contract to the emulator, you can run the following command:
 
 ```
-// script.cdc
+flow project deploy
+```
 
-import "HelloWorld"
+The contract will now have been deployed to the account you selected earlier. You can now interact with it with a script.
```
Comment (gregsantos) [inline docs/build/getting-started/quickstarts/flow-cli.md:92]:
```suggestion
The contract will now have been deployed to the account you selected earlier. You can now interact with it using a script.
```

File: docs/build/getting-started/quickstarts/flow-cli.md [hunk 4]
```diff
@@ -175,87 +71,58 @@ access(all) contract HelloWorld {
...
 
-access(all)
-fun main(): String {
-  return HelloWorld.greeting
-}
+## Running Scripts
 
-```
+On Flow, scripts are used to read data from the Flow blockchain. There is no state modification. In our case, we are going to read a greeting from the `HelloWorld` contract.
```
Comment (gregsantos) [inline docs/build/getting-started/quickstarts/flow-cli.md:96]:
```suggestion
Scripts are used to read data from the Flow blockchain. There is no state modification. In our case, we are going to read a greeting from the `HelloWorld` contract.
```

**Review by gregsantos — APPROVED** [review]
few suggestions but lg2G

**Pull request merged**